packages/lib-async-states/package.json

```diff
@@ -19,9 +19,8 @@
     "_test": "mocha",
     "build": "npm run _clean && npm run _compress",
     "lint": "standard src --fix | snazzy; exit 0",
-    "prepare": "npm run build",
```

Comment:
Why is this being removed?

Reply:
Because I've noticed lerna also picks this up and runs it on bootstrap, and I want to minimise the amount of magic happening.

Reply:
Once we're ready to start publishing packages regularly, we can review it and get a procedure in place, but I don't think we're there yet.